project-base/config/services.yaml

```diff
@@ -96,3 +96,16 @@ services:
     App\Controller\Front\ErrorController:
         arguments:
             $environment: '%kernel.environment%'
+
+    Symfony\Bridge\PsrHttpMessage\Factory\DiactorosFactory: ~
+
+    sensio_framework_extra.psr7.http_message_factory:
+        class: Symfony\Bridge\PsrHttpMessage\Factory\DiactorosFactory
+
+    Psr\Http\Message\ServerRequestInterface:
+        factory: ['@Symfony\Bridge\PsrHttpMessage\Factory\DiactorosFactory', 'createRequest']
+        arguments: ['@=service("request_stack").getMasterRequest()']
+
+    App\Controller\Front\RobotsController:
+        arguments:
```

Comment:
I'm a bit lost here. can you please explain why it was needed?

Reply:
Really, I don't know. Maybe would be better if someone else look at symfony4 autowiring and after we can discuss about it. I did changes in services becouse I had errors with indefined services.

Reply:
I suggest contacting @TomasLudvik

Reply:
You can see commit a18654b1b420ccf089c486b948bd63a4b3e2c1fa and some news

Reply:
But in services.yaml it was trap during updated framework-extra-bundle - a104145d3ee9c9dfd166d5858df7e6d76c637015